README.md

```diff
@@ -112,7 +112,7 @@ NOTE: `size` must be an even number.
 ## Credits
 
 Niceware was inspired by
-[Diceware](http://world.std.com/~reinhold/diceware.html). Its wordlist is
```

Comment:
i think you meant to replace the link below this

Reply:
d'oh - will correct later